br/pkg/lightning/config/config.go

```diff
@@ -1066,6 +1068,13 @@ func (t *TikvImporter) adjust() error {
 		return common.ErrInvalidConfig.GenWithStack("tikv-importer.backend must not be empty!")
 	}
 	t.Backend = strings.ToLower(t.Backend)
+	if !t.ParallelImport && t.IncrementalImport {
+		// need to assign t.IncrementalImport to t.ParallelImport when t.ParallelImport is false and t.IncrementalImport is true
+		t.ParallelImport = t.IncrementalImport
+	} else if t.ParallelImport && !t.IncrementalImport {
+		// need to assign t.ParallelImport to t.IncrementalImport when t.IncrementalImport is false and t.ParallelImport is true, to keep compatibility
+		t.IncrementalImport = t.ParallelImport
```

Comment:
because IncrementalImport is deprecated, we don't need to update it anymore

Reply:
right, modified